src/client/updater.py

```diff
@@ -141,13 +155,9 @@ def showSettings(self):
 @with_logger
 class UpdateChecker(QObject):
     gh_releases_url = Settings.persisted_property('updater/gh_release_url', type=str, default_value='https://api.github.com/repos/FAForever/client/releases?per_page=20')
+    updater_downgrade = Settings.persisted_property('updater/downgrade', type=bool, default_value=False)
```

Comment:
persisted_property is just a wrapper around Settings.get and Settings.set so yes, this is fine.